test/Prototypes/DoubleWidth.swift.gyb

```diff
@@ -247,22 +225,18 @@ extension DoubleWidth: _ExpressibleByBuiltinIntegerLiteral {
 
 
 extension DoubleWidth : FixedWidthInteger {
-  @_fixed_layout // FIXME(sil-serialize-all)
-  public struct Words : Collection {
+    public struct Words : Collection {
```

Comment:
Something's gone wrong with the indentation here.

Reply:
Thanks for the spot! This is why we flatten declarations, so I did that.